pkgs/development/compilers/cudatoolkit/hooks/mark-for-cudatoolkit-root-hook.sh

```diff
@@ -1,8 +1,14 @@
 # shellcheck shell=bash
 
+# Should we mimick cc-wrapper's "hygiene"?
+[[ -z ${strictDeps-} ]] || (( "$hostOffset" < 0 )) || return 0
```

Comment:
I shouldn't backport this bit though

Reply:
@SomeoneSerge why not?

Reply:
it's potentially breaking behavior change  where this logic ran regardless before.

Reply:
I attached the comment at the wrong file btw, This one only affects `cudaPackages.markForCudatoolkitRootHook` which is used internally in `cudaPackages`, so it's OK to keep it. I had commented out the guard in the `setupCudaHook` already